addons/t9n/static/src/web/open_resource_list_action.js

```diff
@@ -0,0 +1,13 @@
+import { Component, xml } from "@odoo/owl";
+
+import { ResourceList } from "@t9n/core/resource_list";
+import { registry } from "@web/core/registry";
+import { standardActionServiceProps } from "@web/webclient/actions/action_service";
+
+export class OpenResourceList extends Component {
+    static components = { ResourceList };
+    static props = { ...standardActionServiceProps };
+    static template = xml`<ResourceList/>`;
+}
+
+registry.category("actions").add("t9n.open_resource_list", OpenResourceList);
```

Comment:
Instead of defining a client action for each view, you could have only one for the root, and add a field to the root component to determine which view to display. Example:
```js
export class App extends Component {
    static components = { ProjectList, ResourceList };
    static props = {};
    static template = "t9n.App";

    currentView = "project-list";
}
```
```xml
<t t-name="t9n.App">
    <ProjectList t-if="currentView === 'project-list'"/>
    <ResourceList t-if="currentView === 'resource-list'"/>
</t>
```